compiler/noirc_evaluator/src/ssa/ir/instruction.rs

```diff
@@ -92,6 +93,7 @@ impl std::fmt::Display for Intrinsic {
             Intrinsic::AsField => write!(f, "as_field"),
             Intrinsic::AsWitness => write!(f, "as_witness"),
             Intrinsic::IsUnconstrained => write!(f, "is_unconstrained"),
+            Intrinsic::DeriveGenerators => write!(f, "derive_pedersen_generators"),
```

Comment:
The enum variant name hasn't been updated here.

Reply:
Done